code/datums/quirks/negative_quirks/addict.dm

```diff
@@ -17,17 +11,19 @@
 	var/obj/item/accessory_type //! If this is null, an accessory won't be spawned.
 	var/process_interval = 30 SECONDS //! how frequently the quirk processes
 	var/next_process = 0 //! ticker for processing
-	var/drug_flavour_text = "Better hope you don't run out..."
+	var/drug_flavour_text = "Better hope you don't run out... of what, exactly? You don't know."
+	abstract_parent_type = /datum/quirk/item_quirk/addict
```

Comment:
Please put modifications to existing vars above var declarations.

Reply:
I'm not sure what you mean by this?

Reply:
Things prefixed by `var/` are declarations of new vars that you are creating on this subtype.
Things not prefixed by `var` are not.
Please put all new things below all old things.